dts/bindings/i2c/st,stm32-i2c-v2.yaml

```diff
@@ -13,3 +13,13 @@ properties:
 
     interrupts:
       required: true
+    # Precise timings values for a given Hardware can be pre-computed with a tool like STM32CubeMX
+    # or directly from I2C_TIMING register description.
+    # Because timing value is valid for a given I2C peripheral clock frequency and target I2C bus speed,
+    # each timing value must be provided with the matching configuration.
+    # The resulting table entries should look like <periph_clock i2c_speed_kHz timing>
```

Comment:
The `I2C_BITRATE_` devicetree macros specify speed in Hz, not kHz.

Reply:
updated in last PR version